### DISCUSSION TITLE
Websocket & CaptivePortal

### DISCUSSION THREAD

**Discussion by Koxx3** [Hieromon/AutoConnect#328] [Q&A] [answered]
Hello,
do you have any example with bi-directinnal websocket working wihtout internet access (captive portal) ?
Thanks !
F.

**Comment by Hieromon** [comment, marked as answer]
If you are supposed to use the contributed library, as far as I know, you have to use the [ESPAsyncWebServer](https://github.com/me-no-dev/ESPAsyncWebServer).

**Reply by Koxx3** [reply]
ok ;)
